lang_GENERIC/analyze/Naming_AST.ml

```diff
@@ -136,8 +136,17 @@ module V = Visitor_AST
 type resolved_name = AST_generic.resolved_name
 (*e: type [[Naming_AST.resolved_name]] *)
 
+(*s: type [[Naming_AST.entinfo]] *)
```

Comment:
note that you don't need to add those special s: e: comment. They are actually autogenerated from the tool I use
to generate also the documentation. See https://github.com/aryx/syncweb
The only think I ask you is to not remove the one that are already there